README.md

```diff
@@ -78,9 +74,14 @@ To install ipython/jupyter dependencies for the examples:
 
     pip install -r data_api/requirements-jupyter.txt
```

Comment:
We should add a warning about how long this will take:

**Note**: This has to install `numpy` and other potentially large packages, and may take tens of minutes to complete.